GameData/RP-1/Tree/ProceduralAvionics.cfg

```diff
@@ -105,7 +105,7 @@ AVIONICSCONFIGS
 	AVIONICSCONFIG
 	{
 		name = Near-Earth
-		description = This part provides full control of the craft, instead of the limited control from Science core. The mass and volume of avionics necessary depend on tech level and the mass of the whole craft. It is functional only when close to Earth (up to 2x GEO distance). Uses: LV ascent guidance and upper stages that operate around the Earth.
+		description = This part provides full control of the craft, instead of the limited control from Science core. The mass and volume of avionics necessary depend on tech level and the mass of the whole craft. It is functional only when close to Earth (up to 2x GEO distance) due to a lack of shielding. Uses: LV ascent guidance and upper stages that operate around the Earth.
```

Comment:
Shielding is only part of the reason. There are others like star trackers etc so IMO shielding shouldn't singled out here.